website/content/docs/architecture/scale.mdx

```diff
@@ -0,0 +1,283 @@
+---
+layout: docs
+page_title: Recommendations for operating Consul at scale
+description: >-  
+  When using Consul for large scale deployments, you can ensure network resilience by tailoring your network to your needs. Learn more about HashiCorp's recommendations for deploying Consul at scale.
+---
+
+# Operating Consul at Scale
+
+This page describes how Consul's architecture and impact its performance with large scale deployments, and shares recommendations for operating Consul in production at scale.
+
+## Overview
+
+Consul is a distributed service networking system deployed as a centralized set of servers that coordinate network activity using sidecars that are located alongside user workloads. When Consul is used for its service mesh capabilities, servers also generate configurations for Envoy proxies that run alongside service instances. These proxies support service mesh capabilities like end-to-end TLS and progressive deployments.
```

Comment:
Just noticed that we don't introduce the term data plane and control plane anywhere, can we write one sentence here about each since further below we have a section titled data plane resiliency